spec/lib/popular_article_spec.rb

```diff
@@ -0,0 +1,60 @@
+require File.join(Rails.root, 'lib', 'popular_article')
+
+describe PopularArticle do
+  let!(:blog) { create(:blog) }
+
+  subject { described_class }
+
+  describe '#find' do
+    before(:each) do
+      allow(GoogleAnalytics::API).to receive(:fetch_article_page_views).and_return(popular_articles)
+    end
+
+    let(:popular_articles) {
```

Comment:
use do/end for multiline blocks https://github.com/bbatsov/ruby-style-guide#single-line-blocks